src/Network/Mail/Postie/Protocol.hs

```diff
@@ -1,3 +1,5 @@
+{-# OPTIONS -fmax-pmcheck-models=100 #-}
```

Comment:
gets rid of a warning that the typechecker wasn't able to exhaustively check the pattern match on the state machine transition